lib/ejs.js

```diff
@@ -344,8 +345,13 @@ exports.renderFile = function () {
   }
   opts.filename = path;
 
+  if (!opts.context) {
+      opts.context = this;
```

Comment:
We actually shouldn't be setting props directly on the passed-in opts object -- we should be using the `shallowCopy` utils method here. But this is a preexisting problem; I'll fix it in a subsequent commit.